examples/hello-debian/README.md

```diff
@@ -0,0 +1,16 @@
+# LaunchDarkly Lua server-side SDK Debian Bookworm example
+
+We've built a minimal dockerized example of using the Lua SDK within a Debian Bookworm Docker container. 
+
+## Build instructions
+
+1. On the command line from the **root** of the repo, build the image from this directory with `docker build -t hello-debian -f ./examples/hello-debian/Dockerfile .`.
```

Comment:
I think this is the wrong command for "this" directory.

Reply:
I think it needs to be built from the root of the repository.

Reply:
It does:
> 1. On the command line from the **root** of the repo,

Reply:
Oh, maybe "build the image from this directory" is awkward to parse